app/core/utils.coffee

```diff
@@ -250,6 +250,11 @@ getByPath = (target, path) ->
 
 isID = (id) -> _.isString(id) and id.length is 24 and id.match(/[a-f0-9]/gi)?.length is 24
 
+isIE = ->
+  return false unless navigator? # Only relevant on client-side
+  # http://stackoverflow.com/questions/19999388/jquery-check-if-user-is-using-ie
+  navigator.userAgent.indexOf('MSIE') > 0 or !!navigator.userAgent.match(/Trident.*rv\:11\./)
```

Comment:
We could also use the https://www.npmjs.com/package/jquery.browser plugin we have included, where this would just be `$.browser.msie`.

Reply:
Ah, will look into using that instead.